webview-ui/package.json

```diff
@@ -3,13 +3,11 @@
 	"version": "0.1.0",
 	"private": true,
 	"dependencies": {
-		"@testing-library/jest-dom": "^5.17.0",
-		"@testing-library/react": "^13.4.0",
-		"@testing-library/user-event": "^13.5.0",
 		"@types/jest": "^27.5.2",
-		"@types/node": "^16.18.101",
+		"@types/node": "^20.x",
 		"@types/react": "^18.3.3",
 		"@types/react-dom": "^18.3.0",
+		"@types/vscode-webview": "^1.57.5",
```

Comment:
can you explain why we need this as a regular dependency now?

Reply:
This really should be a dev-dependency.  We don't push this with and public code, as the webview is built.